src/dg/dg.cpp

```diff
@@ -2729,7 +2776,7 @@ void DGBase<dim,real,MeshType>::apply_inverse_global_mass_matrix(
     const FR_Aux_enum FR_Type_Aux = this->all_parameters->flux_reconstruction_aux_type;
      
     const unsigned int init_grid_degree = high_order_grid->fe_system.tensor_degree();
-    OPERATOR::mapping_shape_functions<dim,2*dim> mapping_basis(1, max_degree, init_grid_degree);
+    OPERATOR::mapping_shape_functions<dim,2*dim> mapping_basis(1, init_grid_degree, init_grid_degree);
```

Comment:
I'm not too familiar with this function, so I was wondering why it's changed from `max_degree` to `init_grid_degree`

Reply:
Max degree is for the solution polynomial basis, whereas init grid degree is used for the metric mapping shape function. The mapping shape functions are used to formulate the metric Jacobian etc and all transformations between physical and reference spaces.

Reply:
Ah I see, thanks for the explanation